azext_edge/edge/providers/checks.py

```diff
@@ -71,7 +71,7 @@ def run_checks(
             )
```

Comment:
We discussed the following behavior changes:
  - If AIO is not deployed on the cluster, pre deployment checks run by default.
  - If AIO is deployed on the cluster, pre deployment checks do not run by default.
    - A user can still run pre-deployment checks at anytime using `--pre`.

Reply:
currently, it will run pre all the time. I agree with having pre run only if AIO is not installed or pre is explicitly called.

Will change